### PR TITLE
Update UPGRADE-5.0.md

### PR DESCRIPTION
**Pull request by noniagriconomie**
| Q             | A
| ------------- | ---
| Branch?       | 5.0
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | 
| License       | MIT
| Doc PR        | https://symfony.com/doc/4.4/setup/built_in_web_server.html

Small improvement

### CODE REVIEW

File: UPGRADE-5.0.md [hunk 1]
```diff
@@ -681,4 +681,4 @@ WebProfilerBundle
 WebServerBundle
 ---------------
 
- * The bundle has been removed.
+ * The bundle has been removed. Use the `Symfony Local Web Server` instead.
```
Comment (fabpot) [inline UPGRADE-5.0.md:684]: I would do another change here. The bundle has not been removed, but deprecated in core and extracted.

Comment (noniagriconomie) [reply UPGRADE-5.0.md:684]: @fabpot what about?

```suggestion
 * The bundle has been deprecated and can be installed separatly. You may also use the `Symfony Local Web Server` instead.
```

Comment (fabpot) [reply UPGRADE-5.0.md:684]: Works for me.

typo: separately

File: UPGRADE-5.0.md [hunk 2]
```diff
@@ -681,4 +681,4 @@ WebProfilerBundle
 WebServerBundle
 ---------------
 
- * The bundle has been removed.
+ * The bundle has been deprecated and can be installed separately. You may also use the `Symfony Local Web Server` instead.
```
Comment (nicolas-grekas) [inline UPGRADE-5.0.md:684]: why using backticks?

**Review by nicolas-grekas — COMMENTED** [review]
for 4.4

File: UPGRADE-5.0.md (hunk 2 above)
Comment (noniagriconomie) [reply UPGRADE-5.0.md:684]: indeed useless, a habit on personal doc to highlight some wording :)

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @noniagriconomie.

**Pull request merged**